migration-roadmap/alert-banner.md

```diff
@@ -109,107 +155,132 @@
+| CSS selector                                                                                     | Attribute or slot                       | Status           |
+| ------------------------------------------------------------------------------------------------ | --------------------------------------- | ---------------- |
+| `.spectrum-AlertBanner--info`                                                                    | `variant="info"`                        | Implemented      |
+| `.spectrum-AlertBanner--negative`                                                                | `variant="negative"`                    | Implemented      |
+| `.spectrum-AlertBanner.is-open`                                                                  | `open` attribute                        | Implemented      |
+| `.spectrum-AlertBanner-text`                                                                     | Default slot                            | Implemented      |
+| `.spectrum-AlertBanner-icon`                                                                     | Icon rendering (info/negative variants) | Implemented      |
+| `.spectrum-AlertBanner:has(.spectrum-CloseButton) .spectrum-AlertBanner-body`                    | `dismissible` attribute                 | Implemented      |
+| `.spectrum-AlertBanner:lang(ja), .spectrum-AlertBanner:lang(ko), .spectrum-AlertBanner:lang(zh)` | Language-specific styling               | Implemented      |
+| `.spectrum-AlertBanner`                                                                          | Base component                          | Implemented      |
+| `.spectrum-AlertBanner-body`                                                                     | Internal structure                      | Implemented      |
+| `.spectrum-AlertBanner-content`                                                                  | Internal structure                      | Implemented      |
+|                                                                                                  | `action` slot                           | Missing from CSS |
 
-**Features Being Deprecated/Removed:**
+## Summary of changes
 
-- Divider element between content and close button has been removed in Spectrum 2
+### CSS => SWC changes
 
-### Action Items for Web Component Maintainers
+**No missing features.** All CSS selectors have corresponding web component implementations:
 
-**Required Additions:**
+- **Variants**: `--info` → `variant="info"`, `--negative` → `variant="negative"`
+- **State**: `.is-open` → `open` attribute
+- **Content**: Text and icon selectors → default slot and programmatic icon rendering
+- **Dismissible**: Close button presence → `dismissible` attribute
+- **Action slot**: Available in SWC but not in CSS templates
 
-- No major additions needed - component is well-aligned with CSS implementation
+### CSS DOM structure changes
```

Comment:
To do: the reason why it's valuable to surface this is because it shows what changes we made in our Spectrum 2 migration work (vs. the CSS/SWC section above it, which would ideally show any gaps in implementation between CSS and SWC), and therefore is likely to show any new features we've added for S2. I think a name change, something like "Spectrum 2 changes" would be valuable here.

I think it's probably helpful to update the prompt accordingly.